server/subsonic/media_retrieval.go

```diff
@@ -64,8 +64,9 @@ func (api *Router) GetCoverArt(w http.ResponseWriter, r *http.Request) (*respons
 	p := req.Params(r)
 	id, _ := p.String("id")
 	size := p.IntOr("size", 0)
+	square := p.BoolOr("square", false)
```

Comment:
`"square"` is not a parameter for Subsonic or OpenSubsonic getCoverArt. Probably can just pass false directly

Reply:
Oh... looks like the web UI uses the subsonic api to fetch images?

Reply:
Yeah, I proposed this. This is a quick "hack" to fix an issue in the (current) WebUI. 
I don't mean to document and/or propose this to OpenSubsonic, it is just an internal thing.

For the new UI We will introduce a proper native artwork endpoint, and this parameter will be removed from the `getArtWork` endpoint.